### ISSUE TITLE
What Sub-systems are up for revision?

### ISSUE THREAD

**Issue by Tarkisflux** [SqueeG/awesomeTome#2]
Hello project contributor! Your willingness to contribute your time gratis entitles you to a say in this, and every other open, issue thread for this project. And this is the most broad of them all.

The issue here is simple enough to start with, but really really thorny in the details. We know that we're going to edit some things together and replace some SRD things. Grappling, for example, is almost certainly going to be F&K bits rather than the normal SRD nonsense. Other things are less clear though. We have a chance to make some changes system wide, and we should sort those changes at the outset so we don't invalidate a bunch of work later on.

So, what do you want to see changed, and what would you like to see it changed to? Here's some examples of things that we could alter:
*Resource Management Systems (like spell slots)
*Multiclassing
*Skills
*Feat Progressions
*Equipment and Magic Items
*Alignment
*Combat

If you have anything you really really want tweaked, say so now and make the case for it. You may not get your way on everything, but hopefully the rest of it is something you can get behind and you can just do minor tweaks at your own table. The aim is for your work to be of benefit to you after all, not just for you to make someone else's game for them. And if the ultimate direction is something you really can't get behind, you always have the option of contributing some things and forking the rest out for yourself.

**Comment by saulrh** [comment]
- Equipment and Magic Items. As a state-of-the-world checkup, does anybody know what the state of the Den's item rules is right now? I'm a bit out of the loop, but I know that there's a half-finished Book of Gears somewhere and a few people have put together their own modifications to the crafting rules. Equipment is one of those things that we really should be able to improve on, so I feel like it would be a good idea to collect all of those fixes and hammer them together into something coherent.

**Comment by saulrh** [comment]
- Resource Management Systems (like spell slots). I think that these are baked into individual classes and abilities well enough that we shouldn't (and shouldn't have to) muck with them. Replacing a resource scheme is effectively a complete rewrite, and that's already been done for us.

**Comment by saulrh** [comment]
- XP Management. We probably want to see if we can do something about character levels, challenge rating, encounter level, and XP assignment.

**Comment by SqueeG** [comment]
Personally I'm a fan of what Shatner did with USE.

http://tgdmb.com/viewtopic.php?p=303573#303573

He normalized BAB progression while multiclassing, addressed 'organic' character growth via Early Levelling Oppurtunities and hotfixed skills.

It's not perfect, and I could see tracking rerolls related to specific skills getting cumbersome, but I think it's a good starting point for simplifying skill allocation.

It definitely will not scratch the itch of people who want MOAR COMPLEX skills though. There's a really cool skill system up on the dnd-wiki that addresses that, though.
- Equipment : I think... Red Rob finished his Tome Equipment project. That simplified the idea Frank and K started. a little bit. That might be a good starting point.

In general I think it might be beneficial to find as much drag and drop material as possible. If we were looking to make and edition-leap forward I could see really going elbows-deep in rewriting stuff, but for now just getting a coherent, updated, TomeBook together would be great.

**Comment by Bihlbo** [comment]
I agree that USE is a very good way to handle skills. However, for the project I think it's too much of a change. It would be better to present it as an alternative. And if it is, there's a good reason to expand it to USER because of skill feats. I have a conversion started that basically goes, "Training in a skill past E is Remarkable, for which you make checks just like the E level but you roll 2d20 and pick one. Rank 0 and 4 in skill feats are granted when you get the feat. The higher ranks translate thus: rank 9 requires Skilled and level 5, rank 14 requires Expert and level 10, rank 19 requires Remarkable and level 15." However the R level might be a bit too OP with what amounts to an automatic reroll with every check, and if so you're just going to have to give away that top tier ability on the skill feat based on level.

As for the rest of Shatner's houserules, I have a writeup of how to handle progressions and multiclassing that is better worded and more streamlined, while accomplishing the same thing. I'll get the file up when I can. I'm seriously not a fan of open multiclassing however, and strongly believe that the document aught just to say, "Please, only one base class per character. The game doesn't handle multiclassing well." That's the simplest way to handle it. I mean, it's 3.5 right? People who come from that will use multiclassing rules from 3.5 and you know, whatever. But we aught to at least point out that it's suboptimal.

**Comment by SqueeG** [comment]
- Multi-classing : I don't have a problem explaining that 3e multi-classing is generally a bad idea ( and quickly explaining why ). Or, at least cross-role multi-classing.

I also think we should push PrCs as a better alternative as a "quickfix" kind of thing. But we'd have to make sure there are some GOOD gish choices to PrC with. Because people damn well wan their fireballin' sword stabbers.

**Comment by Tarkisflux** [comment]
In retrospect, this would probably have worked better as a series of issues about specific subsystems to avoid clutter...

Ah well. I'll drop these here for now, so that we can negotiate over multiple items if we want, and we can do issues for specific forms of these things once we've picked general forms. Some of these are more intensive than just grabbing, formatting, and editing though. Which begs the question, "how much design work vs incorporation work do you want to do"? 

<h4>Skills</h4>

<ul><li> I wrote the <a href="http://www.dnd-wiki.org/wiki/Tome_of_Prowess_%283.5e_Sourcebook%29">Tome of Prowess</a> that Glaze is referring to on the wiki, and I don't particularly want to include it here. I'll help you get it in here anyway if you really want it, but there are a lot of other subsystems that I would want to see reworked if it got put in, because coherence. I can make that pitch if anyone is interested, but I think it's probably too much work for this project. It's also not complete (well, the current editing and re-formatting pass isn't anyway, but it's mostly useable otherwise) and I don't want to be holding things up.</li>
<li>I agree that USE is probably too large a deviation for the current scope of the project, but then I'm also not really a fan of it's high level behaviors. Still, if that's what people want and like then that's what we should do. If you want to test the waters of whether it has general appeal or not, I can throw down a Den poll with that as an option.</li>
<li>There's also skill consolidation and possible outsourcing of some things to backgrounds or whatever. I think there's room to remove a few things from the skill point method and write more detailed rules for level independent backgrounds, but that's a bit more work again and not necessary in a consolidation setup. We could just tell people to handwave things we don't include.</li></ul>

<h4>Multiclassing</h4>

Open multiclassing seems right out, which I think we're all fine with. That leaves us...

<ul><li>'No multiclassing, except through PrCs' is certainly an option, and not a bad one. Probably not too much work even.</li>
<li>The full dungeon crusade is an option too, where classes just go 4 or 5 levels and then that's it and you pick a different one. This is a lot more work though, and possibly more different than people want from this project.</li>
<li>I'll throw 2e/gestalt style multiclassing out there as well, where you just get class features from 2 or more classes and pay for it with reduced advancement. I wrote a variant for it on the wiki that you can <a href="http://www.dnd-wiki.org/wiki/Gestalt_Style_Multiclassing_%283.5e_Variant_Rule%29">read here</a> if you want to consider it.</li></ul>

I'd also like to look into treating PrCs more like alternate class features rather than bolt on classes. So if you qualify for a PrC and take it from 6-10, when you get out of it you get level 11 class features in your base class instead of level 6 features. It's sort of the absolutely horrible level selection that JE mentioned a while back, but limited to cases where it actually makes sense and should reduce level appropriate ability lag.

<h4>Equipment</h4>

Surgo took F&K's and Kaelik's work and wrote up a completed <a href="http://www.dnd-wiki.org/wiki/Tome_Magic_Items_%283.5e_Variant_Rule%29">Tome Magic Items</a> rule set. There are some weird bits with spell arrows and potions that I'd want to see addressed, but otherwise we can just grab that.

There are two or three completed RoW style armor revisions floating around. 

<ul><li>The original RoW completed version, that I can't find a link to and don't like anyway.</li>
<li><a href="http://www.tgdmb.com/viewtopic.php?t=52854">Josh Kablack's revision</a>. It's slightly unfinished though, and I think he prefers the next one anyway.</li>
<li><a href="http://www.tgdmb.com/viewtopic.php?t=53563">Red Rob's simplified revision</a>, which I think is probably the best of the bunch.</li></ul>

Any of these are probably easy enough to incorporate.

<h4>Save Scaling</h4>

This hasn't been discussed, but I wanted to throw it out. Does anyone want to put save scaling (both DCs and saving throw progressions) up on a +level progression instead of on a +half level progression? It interacts better with skills and bab and whatever else that way, but it also means that lower level characters have an even worse chance of saving against higher level effects.

**Comment by SqueeG** [comment]
- **As far as work goes** : I'd like to see **this** version be as simple as possible. Just so we can get a "first edition tome" officially put together.

If we would like to **also** work on some serious reworks for a "next edition" in parallel, I'm also game for that.

So redoing the entire base class system and the like would be a TNE kind of thint. Creating PrCs and Alternate Class levels could be for **this** edition.

Just so we can keep the ball rolling.
I think this edition would be best suited as THE next step toward TNE.

Knkw what I mean?

**Comment by ExplosiveRunes** [comment]
I'll divide my opinions up into two sets, things that I'd like to see in the first edition of a completed tome, and things I'd like to see in something like TNE using Tome as its base. I'll just do first edition for now.

_First Edition_
- **Multiclassing** - No real change. Make sure that the "core" classes each have at least two obvious PRCs to go into, and that you can completely fill out levels 6-20 with PRCs if you so desired.
- **Skills** - Keep similar to standard 3.5 while addressing issues. Include errata where reasonable. Reduce the number of skills slightly by combining them. 
- **Equipment** - I think the simplified armors, or something like it, are the way to go. I like the scaling armor, but it adds a layer of unneeded complexity to the game. Having played tome games often, that shit is a bitch to keep on your character sheet. We also need a weapon system, at one point I made one (under the name Utterfail) [over here](http://tgdmb.com/viewtopic.php?t=50839&postdays=0&postorder=asc&start=0) that tried to follow the suggestions in the tome, it seemed to be moderately well received (not hated), so something similar might be okay. As for magic items, I don't even know man.

**Comment by Tarkisflux** [comment]
ExpRunes, we also have the SRD weapons we can just import. I'll take a look at it again in a bit, but could you toss out why yours is an improvement over that?

**Comment by ExplosiveRunes** [comment]
Well, the main purpose is to get rid of the weird weapon sizing edge cases, where by the rules a medium shortsword and a small longsword have the same statistics, but a medium creature attempting to use the small shortsword takes a penalty to attacks for its 'inappropriate' size. Under my system both are just 'small swords', and a small creature has to use it two handed while a medium creature can use it one handed (and a large creature can use it as a light weapon). It also obviates all the bullshit about putting out new large versions of weapons in splatbooks that wizards did "ooh, look, its a _great_ pick".

It's also slightly rebalanced, and I think makes certain weapons more viable, and has more interesting simple weapons (you can treat any martial weapon 3 sizes smaller than you as simple). It would also probably interact with certain feats in a simpler way, combat school could just apply to "swords" or "axes" instead of some indeterminate number of weapons that you can BS your DM to agreeing to.

All of that said, there's nothing particularly godawful about the standard 3.5 or 3e weapon systems, I just created that one because thats how the Tomes themselves recommended doing it and I was trying to fill that in.

**Comment by Lokathor** [comment]
There's a lot here, but I'll say that we should try to focus on fixing things that are really broken, and things that are not as broken we can just put big fat warning labels on. Like, there's absolutely no way to fix open multiclassing, and it's ALMOST usable in the 1-10 level range if you're not a spellcaster. So, instead we just put a paragraph saying "this option will shoot you in the foot, only do it if you're okay with that".

**Comment by saulrh** [comment]
I agree with the limited-changes-and-warning-labels method, like Lokathor and SqueeG. We don't have the time or dedication to make any major changes, so we try to fix things that are unplayable or only need very small tweaks and put giant disclaimers on everything else. So, along those lines, quick summary of this thread:

Things that look feasible:
- RoW equipment, either simple or scaling. Grab it, LaTeX it, drop it in.
- Surgo's Tome Magic Items. Grab it, LaTeX it, drop it in.
- Spot fixes to skills. Some consolidation, some changes to the really broken things (diplomacy).

Things that look like they're infeasible:
- Multiclassing. This gets a warning label.
- Big changes to skills. USE, Tome of Prowess.

Does that all seem accurate?

**Comment by SqueeG** [comment]
Looks good to me. We could just latex the alternate skill systems and compile them in separate PDFs.

So you could have Tome:: Vanilla, Tome + USE, Tome + TarkisSkills (ToP)

**Comment by saulrh** [comment]
Things that have been mentioned but not discussed:
- Feat progressions. I think that RoW-style feats are good enough.
- Alignment. I've seen a few things about "flags" and a few other things, but I haven't seen a completed alternate alignment system. That probably puts it in the "warning labels" category, with some specific issues being the subject of sidebars drawn from the Tome of Necromancy and Races of War.
- Save Scaling. I'm against it; it's a _simple_ change, but would require hunting down and changing every single save DC and save bonus in the entire system and is probably too much work to do for now.

**Comment by SqueeG** [comment]
#### Scaling Feats

I don't see why we wouldn't include them since they are a rather core concept of RoW.

That being said there have been topics posted suggesting nes ones. **_I'd personally like to see them expanded on as well**_

**Comment by saulrh** [comment]
There are plenty of scaling feats floating around on the Wiki and the Den that we should incorporate, mostly from [here](http://dnd-wiki.org/wiki/3.5e_Scaling_Feats) and [here](http://tgdmb.com/viewtopic.php?t=50239).

**Comment by saulrh** [comment]
Oh, spheres too. We want plenty of spheres.

**Comment by Tarkisflux** [comment]
<h3>Scaling Feats</h3>

Some of them could use some rebalancing, and we probably want to write up a lot more skill ones or convert some of the combat ones that we have in abundance.

<h3>Spheres</h3>

I don't think spheres need to be in the TSRD, though they certainly could be. It's a whole other subsystem to write up after all, for all of one or two classes that are covering rather niche conceptual space. I see a lot of conceptual overlap between "class that contracts with demon prince/high angel/elemental lord" for their power and "cleric", so I don't see a strong narrative need for the classes. If they went in as anything other than an addendum I'd want to lean on them for rewritten outsiders in the beastiary; they often have piles of SLAs anyway and this just unifies them a bit.

<h3> Domains </h3>

These have the same limited applicability as spheres (even if a domain paladin gets in I guess), but they're already attached to a class that is somewhat expected. Unless the cleric gets replaced or left out, these are probably in. And since they mostly suck, I think we can do a lot of replacement here, uploading from the wiki.

**Comment by Lokathor** [comment]
My only thing on domains is to, where possible, replace the "+1 caster level with ___ spells" with more interesting abilities (even if they're slightly weaker or whatever) and/or more noticeable abilities, so that clerics of different domains are more clearly different from one another.

**Comment by SqueeG** [comment]
Right, let's get this party started.

### Scope
- Small, minimal original work beyond existing community material.
- Drag and Drop as much coherent Frank and K material as possible into the 3.5 SRD.
- Drag and Drop as much community material as possible into gaps left by Frank and K
- Errata all previous material formatted in those areas to reflect most current iterations.

#### Feats
- Compile and errata scaling feats including possibly creating more skill feats

#### Skills
- Consolidate obvious skills, while dropping others
- Poke at Diplomacy w/ a 10' pole

#### Classes
- Have direct Tome replacements, or original SRD representations for each class, with the possible exception of fighter.
- Flag Multi-Classing
- Ensure there are PrC options to make up for multi-classing

#### Equipment
- Slot in some Tome-inspired community material (?)
  - Red Rob's simplified scaling equipment
  - Surgo's Tome Magic Items

#### Combat
- Merge SRD and F+K Tome rules

## Not talked about yet
- Races
- Magic and spells
  - Flag Alignment (?)

**Comment by Tarkisflux** [comment]
On skills, I'm willing to write rules for 3 or 4 phase skills that are separate from the normal ones (and mostly already have these, they're just overly complicated at present), so long as someone is willing to assist with the writing of the background skill (or secondary skill or whatever they get called) entries.

**Comment by SqueeG** [comment]
I'm willing to help in any way possible, let's get some goals pinned down so we can reach them.

**Comment by Tarkisflux** [comment]
I agree with those goals though, and thought they'd already been settled on for the most part. We just haven't been agreeing on doing specific things so much as throwing out alternatives to achieve them and waiting for agreement to happen. Which is why I threw down an actual plan in the classes thread and planned on assigning that out to some less vocal contributors. A list of 20 classes to update and a pile of skills to transcribe gives us things to work on while other things get hammered out. And when they're done, we should have other stuff read to move on to. We don't need all of our goals pegged down right now, just enough to do things other than discussion while the discussion finishes up.

But if you want a list of tasks you can do right now, here you go:

<ul><li>Apply errata to classes that we already have, and move them into new directory when completed. High priority for base 20, moderate priority otherwise.</li>
<li>Transcribe any community class you consider to be well formed, and move them into new directory when completed. Low priority.</i>
<li>Split feats out from current document into their own individual documents, applying errata while splitting. Move them to new directory when completed. Moderate priority.</li>
<li>Update the existing character creation guidelines, or write new ones from scratch. Move to new directory when completed. High priority.</li>
<li>Compile the various alignment /morality discussions from Tome of Necromancy and Races of War, and merge with SRD section (not sure if Lokathor did that part yet). Low priority.</li>
<li>Merge combat with SRD section, including rules on positioning and movement. Advanced combat just becomes advanced maneuvers. Split file up into multiple files based on section. High or moderate priority.</li></ul>

Since Lokathor has the skills in LaTeX already, we don't need to transcribe them so much as settle on a skill list, merge down to it, and then do whatever secondary skill thing we wanted to do. That still needs to be sorted in the skills issue though, so it's not included above.

And we have a lot of spell transcriptions to do as well. But there's plenty there to keep us busy while we work out the small details.

**Comment by ExplosiveRunes** [comment]
The version of the Combat chapter in the awesomeTome is already merged Tome/SRD

**Comment by Tarkisflux** [comment]
True enough. The tone of that chapter is all wrong and includes references to material that don't need to be there though (iterative comments, for example). It needs to be edited pretty bad, even if it is all present already.

**Comment by ExplosiveRunes** [comment]
If you only include combat.tex and combat2.tex you end up with a chapter that looks almost exactly like the SRD with the tome rules (but not ramblings and asides and mass combat) inserted where they need to be, including things like Full Attack being edited to have all iterative attacks at -5, and tome combat maneuvers.  I just compiled a version of the pdf with combat and combat2 as a single chapter, and it looks fine to me, though I suppose some of the content could be rearranged for easier flow for first time readers (maybe the combat maneuvers could come immediately after actions?)

**Comment by SqueeG** [comment]
I assume our scope is somewhere along the lines of what Sigil said the other day?

"We, for the most part, know what we need to do to simply complete the pdf srd style, but we're throwing out big ideas for when we reach that point and go beyond it."

If that's pretty much the gist of it, let's get that up in the wiki and close this thread, then tag it for reference.

**Comment by Tarkisflux** [comment]
Sure. This is a "we don't want major changes, we just want everything centralized into 1 pdf (possibly 3 if there's a separate mcg or beastiary from the phb)" project. Large scale revisions are off the table, but small deviations may still be included.. Issue closed.
